ods_ci/tests/Resources/Files/accelerator/qe_accelerator_profile_2.yaml

```
@@ -0,0 +1,17 @@
+# File to provision an Accelerator Profile instance in test:
+#   "Modify An Accelerator Profile Using "Accelerator Profiles" Administration UI"
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  name: qeacceleratorprofile2
+  namespace: redhat-ods-applications
+spec:
+  description: Modify Accelerator Profile
+  displayName: qe_accelerator_profile_2
```

Just an example, but relevant for all yamls:
```suggestion
  displayName: Test: Modify Accelerator Profile 2
```
This is more clear than with underscores + you test `:` which is a special character (even more than `_`)